src/components/cluster/detail/__tests__/tabs.js

```diff
@@ -22,12 +23,9 @@ it('renders only the first tab', () => {
       <Tab eventKey={3} title='third'>
         <h1>Third Tab</h1>
       </Tab>
-    </Tabs>
+    </Tabs>,
+    div
   );
 
-  expect(tabs.find('#tabs-pane-1').hasClass('active')).toEqual(true);
-
-  expect(tabs.find('#tabs-pane-2').hasClass('active')).toEqual(false);
```

Comment:
I think part of it is also making sure the other panes are not visible. Not sure if now this test still ensures that.

Reply:
True, I assumed incorrectly that they were `position: absolute` and that if the first one was visible, the others will remain not visible. I am correcting it